VERSION

```diff
@@ -1 +1 @@
-v0.10.7-dev
\ No newline at end of file
+v0.10.11-dev
```

Comment:
Just as an additional remark. We let the CI/CD pipeline increment the version. Generally, it will increment the minor release so `v0.1.0-dev` would become `v0.2.0-dev` after `v0.1.0` is released. For patch version increments also happen automatically in special `release-*` branches. For that to happen you should create a special `refs/meta/ci` branch. We could do this later, just writing here as a reminder.